react/week2/todoapp/src/todo-app.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import Timer from './Timer'
+
+function TodosList({ id, description, removeTodo }) {
+    const [isDone, setIsDone] = useState('');
+    return (
+        <div>
+            <li className="list-of-todos" style={isDone ? { textDecoration: "line-through" } : { textDecoration: "" }}>{description}
+                <span>
+                    <input type="checkbox" onClick={() => setIsDone(!isDone)} />
+                    <button onClick={() => removeTodo(id)}>Delete</button>
+                </span>
+            </li>
+        </div>
+    )
+}
+
+const RenderedTodos = ({ todos }) => {
+    const [addTodo, setAddTodo] = useState(todos);
+    const [value, setValue] = useState('');
+    //adding the todos to todos array
+    const OnClickAddTodo = () => {
+        if (value !== "") {
+            const myNextTodos = { id: addTodo.length + 1, description: value };
+            const myNextTodoNames = addTodo.concat(myNextTodos);
```

Comment:
Here addTodo is ur list of todos but the name of variable mean something else .Instead of addTodo you can use todoLists because addTodo variable means you are adding new todo. Write use friendly variable name.

Adding is working here but instead of concat use spread operator.
like [...addTodo, myNextTodos]